components/tools/OmeroPy/src/omero/__init__.py

```diff
@@ -104,6 +104,14 @@ def proxy_to_instance(proxy_string, default=None):
     """
     import omero
     parts = proxy_string.split(":")
+    # If a default is provided and the format is Class:ID
```

Comment:
Guess the question is *is* it the wrong object? If the user requests "Image" then sure, if the method doesn't work on Image, through. But that may be a cli-level decision and not a proxy_to_string level-decision.